packages/paste-core/utilities/grid/src/helpers.tsx

```diff
@@ -0,0 +1,71 @@
+import {ThemeShape} from '@twilio-paste/theme-tokens';
+import {ResponsiveValue} from 'styled-system';
+import {Margin, Space, SpaceOptions} from '@twilio-paste/style-props';
+import {Vertical} from '@twilio-paste/flex';
+import {
+  ColumnOffset,
+  ColumnOffsetOptions,
+  ColumnProps,
+  ColumnSpanOptions,
+  ColumnMinWidth,
+  ColumnWidthSpan,
+} from './types';
+
+export const getOuterGutterPull = (theme: ThemeShape, gutter?: Space): Margin => {
```

Comment:
Let's give these functions some comments now that they aren't in proximity of where they're being used.

Reply:
@TheSisb comments added.